modules/installation-osp-creating-sr-iov-compute-machines.adoc

```diff
@@ -85,15 +84,15 @@ After standing up the control plane, create compute machines that run on the SR-
   - name: 'List the Compute Trunks'
     command:
       cmd: "openstack network trunk list"
-    when: os_networking_type == "Kuryr"
```

Comment:
These two steps with "Kuryr" should be removed, so this whole fragment needs to be gone:

```
  - name: 'List the Compute Trunks'
    command:
      cmd: "openstack network trunk list"
    when: os_networking_type == "Kuryr"
    register: compute_trunks
  - name: 'Create the Compute trunks'
    command:
      cmd: "openstack network trunk create --parent-port {{ item.1.id }} {{ os_compute_trunk_name }}-{{ item.0 }}"
    with_indexed_items: "{{ ports.results }}"
    when:
    - os_networking_type == "Kuryr"
    - "os_compute_trunk_name|string not in compute_trunks.stdout"
```